contents/blog/data-lineage-meetup/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Happening Soon - Our First Meetup!
+date: 2023-02-08
+author: Michael Robinson
+image: ./image.svg
+banner: ./banner.svg
+description: The inaugural Data Lineage Meetup will take place on March 9th in Providence, RI.
+---
+
+### Where, When & How
+
+Date: March 9, 2023  
+Time: 6-8 pm ET  
+Address: [CIC](https://cic.com/providence), [225 Dyer Street](https://goo.gl/maps/NYBbs4ht91dWWaRs9)  
+City: Providence, RI, US 02903  
+Format: In-person
+
+#### Getting There
+- Air: the nearest airport is T.F. Green/PVD. Boston Logan is also within 1.5-2 hours' driving distance.  
+- Rail: Amtrak serves [PVD](https://goo.gl/maps/eAMNPcpQVJkqwyS76), which is within walking distance of CIC.  
+- Road: garages and lots are a short walk away from the venue, and metered street parking is also available nearby.
+    - [Richmond Garage](https://goo.gl/maps/nG8j8Vk6ko75GXaH9) 
+    - [South Street Landing garage](https://goo.gl/maps/QYrbo1oaiU4J3fi69) 
+    - [Clifford parking lot](https://goo.gl/maps/qHoLkRcGmWpxeHhJ8) 
+
+#### Getting In 
+- Check in with the concierge at CIC. They will direct you to the Hope Island Room on the 3rd floor.
+
+### Who & What
+
+Imagine a world where data and jobs are described predictably and consistently. What would you be able to accomplish when operational lineage information travels easily among all your tools?
+
+OpenLineage is an emerging open standard for data lineage that attempts to create that world.
```

Comment:
I think these two paragraphs are not necessary for the blog post on openlineage.io.